doc/radosgw/lua-scripting.rst

```diff
@@ -0,0 +1,338 @@
+=============
+Lua Scripting
+=============
+
+.. versionadded:: Pacific
+
+.. contents::
+
+This feature allows users to provide a Lua script that will be executed after a request has been processed.
+The script will have access to fields in the request, and may also modify some fields. All Lua language features could be used in the script.
+
+
+.. toctree::
+   :maxdepth: 1
+
+
+Script Management via CLI
+-------------------------
+
+Each user may have one script which is executed in the context of the S3 requests of that user.
```

Comment:
Suggested change:

Each user can upload a single script, which runs on the user's S3 requests.